client/lib/products-values/index.js

```diff
@@ -253,6 +255,14 @@ export function isBiennially( rawProduct ) {
 	return parseInt( product.bill_period, 10 ) === PLAN_BIENNIAL_PERIOD;
 }
 
+export function isMonthlyFromSlug( productSlug ) {
```

Comment:
Since `isMonthly()` works for all products while this function works for only Jetpack products, I think this should either be renamed `isMonthlyJetpackProductFromSlug()` or be updated to work with all products to keep consistency in the API here.